libr/reg/reg.c

```diff
@@ -127,6 +127,9 @@ R_API int r_reg_type_by_name(const char *str) {
 	if (!strcmp (str, "all")) {
 		return R_REG_TYPE_ALL;
 	}
+	if (!strcmp (str, "xmm")) {
```

Comment:
xmm type was deprecated because it was intel-specific, and the new name is vec128.

We can shorten the name from vec128 to v128 imho? what do you think?

<img width="1121" alt="Screenshot 2023-01-02 at 01 43 09" src="https://user-images.githubusercontent.com/6431515/210188677-08387440-6a90-441f-ba45-fa453965dd86.png">

Reply:
The version of QEMU I am using (7.2.50) responds with "xmm" names when I enable the gdb stub. So, is it possible we can have a mapping of old names into the newer ones?

Reply:
Its the gdb plugin that should be mapping those names